$MirOS: ports/print/cups/patches/patch-backend_betest_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- backend/betest.c.orig	Sat Jul 12 00:48:26 2008
+++ backend/betest.c	Tue Oct 21 22:56:22 2008
@@ -67,7 +67,7 @@ main(int  argc,		/* I - Number of comman
   */
 
   execl(backend, argv[1], argv[2], argv[3], argv[4], argv[5], argv[6], argv[7],
-        NULL);
+        (char *)NULL);
 
   return (1);
 }
